orc8r/cloud/go/obsidian/swagger/spec_servicer_test.go

```diff
@@ -66,6 +76,66 @@ func TestSpecServicer_NewSpecServicerFromFile(t *testing.T) {
 	assert.Equal(t, testStandaloneFileContents, standaloneRes.SwaggerSpec)
 }
 
+func TestSpecServicer_NewSpecServicerFromSpecsPartialErr(t *testing.T) {
+	service := "a service"
+
+	partialErrMock := &mocks.Specs{}
+	partialErrMock.On(
+		"GetPartialSpec", service,
+	).Return(
+		"", errors.New("partial err"),
+	)
+	standaloneSpec := "standalone spec"
+	partialErrMock.On(
+		"GetStandaloneSpec", service,
+	).Return(
+		standaloneSpec, nil,
+	)
+
+	servicer := swagger.NewSpecServicerFromSpecs(partialErrMock, service)
+	assert.NotNil(t, servicer)
+
+	partialRes, err := servicer.GetPartialSpec(
+		context.Background(), &protos.PartialSpecRequest{})
+	assert.NoError(t, err)
+	assert.Empty(t, partialRes.SwaggerSpec)
+
+	standaloneRes, err := servicer.GetStandaloneSpec(
+		context.Background(), &protos.StandaloneSpecRequest{})
+	assert.NoError(t, err)
+	assert.Equal(t, standaloneSpec, standaloneRes.SwaggerSpec)
+}
+
+func TestSpecServicer_NewSpecServicerFromSpecsStandaloneErr(t *testing.T) {
+	service := "another service"
+
+	standaloneErrMock := &mocks.Specs{}
+	partialSpec := "partial spec"
+	standaloneErrMock.On(
+		"GetPartialSpec", service,
```

Comment:
Similar to above: for constructing these mocks, break lines following this pattern https://github.com/magma/magma/blob/60df24fd4470e09dd02ab03c5429946010ad61eb/orc8r/cloud/go/blobstore/sql_test.go#L122-L124